jsonnet/thanos-mixin/dashboards/rule.libsonnet

```diff
@@ -0,0 +1,162 @@
+local g = import '../lib/thanos-grafana-builder/builder.libsonnet';
+
+{
+  grafanaDashboards+:: {
+    'rule.json':
```

Comment:
I haven't really looked/reviewed this file as we don't have that component in the mixin yet. But we can merge it and revisited once we add the Thanos ruler. :slightly_smiling_face: